src/test/scala/group/scala/karazin/circe/literal/extras/generators/JsonGenerators.scala

```diff
@@ -0,0 +1,21 @@
+package group.scala.karazin.circe.literal.extras.generators
+
+import cats.implicits._
+import io.circe.syntax._
+import io.circe.testing.instances.arbitraryJsonObject
+import io.circe.{Json, JsonNumber, JsonObject, parser}
+import org.scalacheck.Gen
+import org.scalacheck.Arbitrary
+import org.scalacheck.Prop._
+import io.circe.Encoder.encodeJsonObject
+
+/**
+  * as an option, we can put all our json generation methods (incl. custom ones if need be)
+  * in a separate file, like here
+  */
+object JsonGenerators {
+
+  val genJsonObjectDefault: Gen[JsonObject] =
```

Comment:
For ScalaCheck we need given (aka implicit) arbitraries.
Please review the existing test.
What you need here is to add a single line:
```
given Arbitrary[JsonObject] = arbitraryJsonObject
```

Please review 
https://dotty.epfl.ch/docs/reference/contextual/givens.html
https://dotty.epfl.ch/docs/reference/contextual/using-clauses.html

Reply:
If that's our approach, no problem.